libraries/protocol/include/golos/protocol/config.hpp

```diff
@@ -135,7 +135,6 @@
 #define GOLOS_MIN_DELEGATION_MULTIPLIER             10
 
 #define STEEMIT_DEFAULT_DELEGATED_VESTING_INTEREST_RATE (25*STEEMIT_1_PERCENT) // 25%
-#define STEEMIT_MAX_DELEGATED_VESTING_INTEREST_RATE     (80*STEEMIT_1_PERCENT) // 80%
 
```

Comment:
Rename to `STEEMIT_MAX_DELEGATED_VESTING_INTEREST_RATE_PRE_HF19`
Add a new macro `#define STEEMIT_MAX_DELEGATED_VESTING_INTEREST_RATE STEEMIT_100_PERCENT`